vNext/src/NuGet/Microsoft.Orleans.OrleansSQLUtils.nuspec

```diff
@@ -0,0 +1,40 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.Orleans.OrleansSqlUtils</id>
+    <version>$version$</version>
+    <title>icrosoft Orleans Sql Utilities</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft,Orleans</owners>
+    <projectUrl>https://github.com/dotnet/Orleans</projectUrl>
+    <licenseUrl>https://github.com/dotnet/Orleans#license</licenseUrl>
+    <iconUrl>https://raw.githubusercontent.com/dotnet/orleans/gh-pages/assets/logo_128.png</iconUrl>
+    <summary>
+      Sql Utilities Library of Microsoft Orleans - OrleansSqlUtils.dll
+    </summary>
+    <description>
+      Library of utility types for relational storage of Microsoft Orleans.
+    </description>
+    <copyright>Copyright Microsoft 2015</copyright>
+    <tags>Orleans Cloud-Computing Actor-Model Actors Distributed-Systems C# .NET</tags>
+    <dependencies>
+      <group targetFramework=".NETStandard1.5">
+        <dependency id="Microsoft.Orleans.Core" version="$version$" />
+        <dependency id="Microsoft.Orleans.OrleansRuntime" version="$version$" />
+        <dependency id="Microsoft.Orleans.OrleansProviders" version="$version$" />
+        <dependency id="System.Data.Common" version="4.3.0" />
+        <dependency id="System.Reflection" version="4.3.0" />
+        <dependency id="System.Xml.XmlSerializer" version="4.3.0" />
+      </group>
```

Comment:
dependency of OrleansRuntime and OrleansProviders is still here. make sure you commit the change you did.